crypto/ui/ui_lib.c

```diff
@@ -843,8 +871,18 @@ int UI_get_result_maxsize(UI_STRING *uis)
 
 int UI_set_result(UI *ui, UI_STRING *uis, const char *result)
 {
-    int l = strlen(result);
+#if 0
+    /*
+     * This is placed here solely to preserve UI_F_UI_SET_RESULT
+     * To be removed for OpenSSL 1.2.0
+     */
+    UIerr(UI_F_UI_SET_RESULT, ERR_R_DISABLED);
```

Comment:
Oh, make update would remove the entry otherwise?  That seems ... risky (but no fault of this pull request).

Reply:
As it stands now, yes.  We should probably have another look at `uitl/mkerr.pl`...